__tests__/app.test.js

```diff
@@ -0,0 +1,29 @@
+const request = require('supertest')
+const app = require('../app')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const {articleData, commentData, topicData, userData} = require('../db/data/test-data')
+
+afterAll(()=> {
+    return db.end()
+});
+
+beforeEach(() => {
+   return seed({articleData, commentData, topicData, userData})
+})
+
+describe('GET /api/topics', ()=> {
+    test('200: responds with an array of topic objects', ()=> {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response) => {
+            response.body.topics.forEach((topic) => {
+                expect(topic).toMatchObject({
+                    slug: expect.any(String),
+                    description: expect.any(String)
+                });
+            });
+        });
+    });
+});
```

Comment:
You could also add a 404 check and functionality to this, to cover what happens with an endpoint like `/api/nonExistentEndpoint`